csharp/tools/ZeroC.Ice.Slice.Tools/ZeroC.Ice.Slice.Tools.targets

```diff
@@ -79,7 +79,6 @@
         <ItemGroup>
             <_SliceCompileNormalized Include="@(SliceCompile)">
                 <OutputDir>$([MSBuild]::NormalizePath('%(SliceCompile.OutputDir)'))</OutputDir>
-                <IncludeDirectories>$([MSBuild]::NormalizePath('%(SliceCompile.IncludeDirectories)'))</IncludeDirectories>
```

Comment:
I added this NormalizePath in a previous PR, which was causing this bug. It returns a string instead of a list and we end-up with `-Ipath1;path2` instead of `-Ipath1 -Ipath2`.